apps/web/src/pages/Pools/BrowsePoolsPage/components/OngoingRecruitmentSection/OngoingRecruitmentSection.tsx

```diff
@@ -984,35 +998,90 @@ const OngoingRecruitmentSection = ({
             "instructions for section with ongoing pool advertisements",
         })}
       </p>
+      <p data-h2-margin="base(x1, 0, x.25, 0)">
+        {intl.formatMessage({
+          defaultMessage: "Select a job stream",
+          id: "dJXjhw",
+          description:
+            "Placeholder for stream filter in browse opportunities form.",
+        })}
+      </p>
       <div data-h2-display="base(flex)">
-        <FormProvider {...methods}>
-          <Select
-            id="quickFilter"
-            name="quickFilter"
-            label={intl.formatMessage({
-              defaultMessage: "Quick filter",
-              id: "8NB+Ay",
-              description: "A label for a quick filter input.",
-            })}
-            nullSelection={intl.formatMessage({
-              defaultMessage: "Select a job stream",
-              id: "dJXjhw",
-              description:
-                "Placeholder for stream filter in browse opportunities form.",
-            })}
-            options={[
-              ...streamsWithAvailablePools.map((stream) => ({
-                value: stream.key,
-                label: stream.label,
-              })),
-            ]}
-            trackUnsaved={false}
-          />
-        </FormProvider>
+        <DropdownMenu.Root>
+          <DropdownMenu.Trigger>
+            <Button
+              color="primary"
+              mode="outline"
+              data-h2-align-items="base(center)"
+              data-h2-display="base(flex)"
+              data-h2-flex-shrink="base(0)"
+              data-h2-gap="base(0, x.25)"
+              data-h2-radius="base(input, 0px, 0px, input)"
+              data-h2-margin-right="base(0)"
+            >
+              <span>
+                {quickFilterStream === "ALL"
+                  ? intl.formatMessage({
+                      defaultMessage: "All",
+                      id: "qQtJDw",
+                      description: "All statuses",
+                    })
+                  : intl.formatMessage(getPoolStream(quickFilterStream))}
+              </span>
+              <ChevronDownIcon
+                data-h2-height="base(1em)"
+                data-h2-width="base(1em)"
+              />
+            </Button>
+          </DropdownMenu.Trigger>
+          <DropdownMenu.Content data-h2-padding="base(0)">
+            {" "}
```

Comment:
Stray space?

Reply:
I think so, removed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6614/commits/6fe41e33677f9a0b40cf3b8026c1bbe195d76b92